syntax = "proto3";

//option java_package = "com.boskokg.flutter_blue_plus";
//option java_outer_classname = "Protos";
//option objc_class_prefix = "Protos";

message ScanSettings {
  int32 android_scan_mode = 1;
  string service_uuids = 2;
}
